src/encoded/docs/extended_description_VariantSample_inheritance_mode_probandonly.html

```diff
@@ -0,0 +1,32 @@
+
+
+<h5>Inheritance mode values are candidate labels based on novocaller, Cmp-Het caller, or GATK genotype assignments.</h5>
+
+<p>This case is labeled as proband-only. Since no other family members are in the analysis, most of the traditional inheritance modes can not be assigned. <b>Inheritance modes that can NOT be assigned on this case include:</b></p>
+<ul><li> de novo calls
+<li> Dominant (Maternal) vs Dominant (Paternal) vs Recessive
+<li> Loss of Heterozygosity
+<li> Phased Compound Heterozygous calls
+</ul>
+
+<p>Variants on the X or Y chromosome are still classified <b>X-linked</b> or <b>Y-linked</b>, even though they can't be called as dominant or recessive or de novo.</p>
+
+<p><b>Compound Heterozygous calls</b>:</p>
+<ul>
+<li> Two heterozygous variants are classified as Compound Heterozygous candidates if they are on the same gene.
+<li> Phasing: This case will only have <b>Unphased</b> Compound Heterozygous candidates, as mother and father genotypes are required to classify a variant as <b>Phased</b> (Mother: 0/1, Father: 0/0, Child 0/1) and (Mother: 0/0, Father: 0/1, Child 0/1).
+<li> <b>Variant severity</b>: A variant is considered severe if it has a SpliceAI Delta Score >0.8 (S), or has a Clinvar interpretation of Pathogenic or Likely Pathogenic (C), or it has a HIGH or MODERATE Impact rating (Splice acceptor variant, Splice donor variant, Stop gained, Frameshift variant, Stop lost, Start lost, Transcript amplification, Inframe insertion, Inframe deletion, Missense, variant, Protein altering variant).
```

Comment:
I don't think the first "or" is necessary here, and for parallelism each option should start similarly. Also, I think there's an extra comma between "Missense" and "variant" towards the end. 

```suggestion
<li> <b>Variant severity</b>: A variant is considered severe if it has a SpliceAI Delta Score >0.8 (S), has a Clinvar interpretation of Pathogenic or Likely Pathogenic (C), or has a HIGH or MODERATE Impact rating (Splice acceptor variant, Splice donor variant, Stop gained, Frameshift variant, Stop lost, Start lost, Transcript amplification, Inframe insertion, Inframe deletion, Missense variant, Protein altering variant).
```